src/components/shared/forms/editUserForm.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useState } from 'react'
+
+import { User, ExpressValidationError } from 'devu-shared-modules'
+
+import { useAppDispatch } from 'redux/hooks'
+import { SET_ALERT } from 'redux/types/active.types'
+
+import TextField from 'components/shared/inputs/textField'
+
+import RequestService from 'services/request.service'
+
+type Props = {
+  user: User
+}
+
+const EditUserForm = ({ user }: Props) => {
+  const [formData, setFormData] = useState<User>(user)
+  const setAlert = useAppDispatch<typeof SET_ALERT>(SET_ALERT)
+
+  const handleUpdatePreferredName = (preferredName: string) => setFormData({ ...formData, preferredName })
+
+  const submit = () => {
+    RequestService.put(`/api/users/${user.id}`, formData)
+      .then((response) => {
+        const message = response.message
+        setAlert({ autoDelete: true, type: 'success', message })
```

Comment:
I think setting an explicit message here might be a bit nicer. Maybe something like `User has been updated`, idk.

Reply:
Also this as a one liner might be a bit cleaner since you don't really _have_ to declare message. Declaring it is a bit slower since it needs to allocate the memory for it just to let it get garbage collected.

Reply:
set a new hard coded message